examples/tts/conf/rad-tts_dec.yaml

```diff
@@ -0,0 +1,277 @@
+name: RadTTS
+sample_rate: 22050
+
+train_dataset: ???
+validation_datasets: ???
+ckpt_path: None
+export_dir: ???
+sup_data_path: ???
+sup_data_types: ["log_mel", "align_prior_matrix", "pitch", "voiced_mask", "p_voiced", "energy"] 
+
+whitelist_path: "nemo_text_processing/text_normalization/en/data/whitelist/tts.tsv"
+
+# LJSpeech stats (per frame), train
+pitch_mean: 212.35873413085938
+pitch_std: 68.52806091308594
+
+# default values from librosa.pyin
+pitch_fmin: 65.40639132514966
+pitch_fmax: 2093.004522404789
+
+# default values for sample_rate=22050
+n_mels: 80
+n_window_size: 1024
+n_window_stride: 256
+n_fft: 1024
+lowfreq: 0
+highfreq: 8000
+window: "hann"
+
+pitch_loss_scale: 0.1
+durs_loss_scale: 0.1
+mel_loss_scale: 1.0
+
+phoneme_dict_path: "scripts/tts_dataset_files/cmudict-0.7b_nv22.07"
+heteronyms_path: "scripts/tts_dataset_files/heteronyms-030921" #"scripts/tts_dataset_files/heteronyms-030921"
+
+model:
+  # aligner
+  target: nemo.collections.tts.models.RadTTSModel
+  bin_loss_start_ratio: 0.2
+  bin_loss_warmup_epochs: 100
+
+  symbols_embedding_dim: 384
+  n_mel_channels: ${n_mels}
+
+  pitch_loss_scale: ${pitch_loss_scale}
+  durs_loss_scale: ${durs_loss_scale}
+  mel_loss_scale: ${mel_loss_scale}
+
+  pitch_mean: ${pitch_mean}
+  pitch_std: ${pitch_std}
+  
+  text_normalizer:
+      _target_: nemo_text_processing.text_normalization.normalize.Normalizer
+      lang: en
+      input_case: cased
+      whitelist: ${whitelist_path}
+
+  text_normalizer_call_kwargs:
+      verbose: false
+      punct_pre_process: true
+      punct_post_process: true
+      
+  train_ds:
+    dataset:
+      _target_: "nemo.collections.tts.torch.data.TTSDataset"
+      manifest_filepath: ${train_dataset}
+      sample_rate: ${sample_rate}
+      sup_data_path: ${sup_data_path}
+      sup_data_types: ${sup_data_types}
+      n_fft: ${n_fft} 
+      win_length: ${n_window_size}
+      hop_length: ${n_window_stride}
+      window: ${window}
+      n_mels: ${n_mels}
+      lowfreq: ${lowfreq}
+      highfreq: ${highfreq}
+      max_duration: null
+      min_duration: 0.1
+      ignore_file: null
+      trim: False
+      pitch_fmin: ${pitch_fmin}
+      pitch_fmax: ${pitch_fmax} 
+      
+      
+      
+      text_tokenizer:
+        _target_: "nemo.collections.tts.torch.tts_tokenizers.EnglishPhonemesTokenizer"
+        punct: True
+        stresses: True
+        chars: True
+        space: ' '
+        silence: null
+        apostrophe: True
+        sep: '|'
+        add_blank_at: null
+        pad_with_space: True
+        g2p:
+          _target_: "nemo.collections.tts.torch.g2ps.EnglishG2p"
+          phoneme_dict: ${phoneme_dict_path}
+          heteronyms: ${heteronyms_path}
+          phoneme_probability: 0.5
+          #adlr_symbol_id_mapper: ${adlr_symbol_id_mapper_path}
+    dataloader_params:
+      drop_last: false
+      shuffle: true
+      batch_size: 8
+      num_workers: 8
+      pin_memory: false
```

Comment:
Could you pls verify `pin_memory`. All other TTS examples make it `true` by default

Reply:
Nice catch. This will actually make it faster. Thanks!